src/test/ui/suggestions/return-bindings.stderr

```diff
@@ -43,6 +43,68 @@ LL ~     let x = Some(1);
 LL +     x
    |
 
-error: aborting due to 3 previous errors
+error[E0308]: mismatched types
+  --> $DIR/return-bindings.rs:20:46
+   |
+LL |       let s: String = if let Some(s) = opt_str {
+   |  ______________________________________________^
+LL | |
+LL | |     } else {
+   | |_____^ expected struct `String`, found `()`
+   |
+help: consider returning the local binding `s`
+   |
+LL ~     let s: String = if let Some(s) = opt_str {
+LL +         s
+LL ~
```

Comment:
I was gonna make a comment about this and then I realized it is the elided `//~ERROR` 😅

Reply:
Yeah, it renders everything up to the `//` on the next line, which is just whitespace...